oauth2client/client.py

```diff
@@ -1784,7 +1784,9 @@ class OAuth2WebServerFlow(Flow):
   """
 
   @util.positional(4)
-  def __init__(self, client_id, client_secret, scope,
+  def __init__(self, client_id,
+               client_secret=None,
+               scope=None,
```

Comment:
You should change `util.positional(4)` if you are no longer having 4 positional arguments.